internal/store/store_test.go

```diff
@@ -338,27 +306,38 @@ func TestLoadBaseMeta(t *testing.T) {
 }
 
 func TestLoadBlockPart(t *testing.T) {
-	bs, db := freshBlockStore()
+	cfg, err := config.ResetTestRoot(t.TempDir(), "blockchain_reactor_test")
+	require.NoError(t, err)
+
+	bs, db := newInMemoryBlockStore()
 	height, index := int64(10), 1
```

Comment:
Pre-existing, but nigh at hand:
```suggestion
	const height, index = 10, 1
```